src/transformers/models/convnext/configuration_convnext.py

```diff
@@ -64,6 +64,8 @@ class ConvNextConfig(PretrainedConfig):
             The initial value for the layer scale.
         drop_path_rate (`float`, *optional*, defaults to 0.0):
             The drop rate for stochastic depth.
+        image_size (`int`, *optional*, defaults to 224):
```

Comment:
Think we can just remove this from the docstring (see https://github.com/huggingface/transformers/pull/21472#discussion_r1118985964)